releasenotes/notes/fix-circular-entanglement-5aadd5adf75c0c13.yaml

```diff
@@ -0,0 +1,16 @@
+fixes:
```

Comment:
is there a performance improvement by porting this code into rust? if so - we can also indicate it in the release notes.

Reply:
There's no performance benefit it only constructing the entanglement, this is a prerequisite so we can fully construct the circuit library circuits in Rust, without needing to go in Python to construct the entanglement in between 🙂